### ISSUE TITLE
Angular template code completion removes trailing quote

### ISSUE THREAD

**Issue by rekna1** [angular/vscode-ng-language-service#1626]
Issue Type: <b>Bug</b>

When I'm using code completion in angular template it's annoying that the trailing quote is removed upon selection of any of the suggestions.

e.g. in the example below I get code completion suggestion on model
[(ngModel)]="model.|" id="name"

after I select a property on model (tried with enter or tab) the trailing quote is removed:
[(ngModel)]="model.Name id="name"

On top of that, after some time, angular language service stops providing code completion, if I change something in settings of plugin (logging level) it works again (probably just because changing this causes the plugin to reinitialize completely.

Extension version: 13.3.0
VS Code version: Code 1.66.1 (8dfae7a5cd50421d10cd99cb873990460525a898, 2022-04-06T14:50:12.141Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vstes627:30244334
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
pythonvsnew555:30457759
vscscmwlcmt:30465135
pynewfile477:30463512

```

</details>

**Comment by atscott** [comment]
Fixed by https://github.com/angular/angular/pull/45582 and included in the release from today

**Comment by atscott** [comment]
@rekna1 This was fixed in 13.3.2. I can reproduce it in 13.3.1 and 13.3.0 but as soon as I upgrade to 13.3.3, I cannot reproduce it.

"also language serice code completion not working all the time" - Please do not combine two reports into a single issue. The investigation here is only focused on the resolution for the trailing quote being removed. There are known issues in 13.3.3 so upgrading to 13.3.4 may resolve this other issue.
